### PR TITLE
1009  change clustering sizes

### PR DESCRIPTION
**Pull request by mmnoo**
## Release Notes

- **New Features**
  - Introduced a new `.env.sample` file for environment variable configuration.
  - Added multiple issue templates for bug reports, feature requests, and general questions.
  - Launched a new header component with user authentication and navigation.
  - Implemented a filter pane for project filtering based on various criteria.
  - Added a metrics pane to display project-related metrics.

- **Improvements**
  - Enhanced the README with installation instructions and mobile testing procedures.
  - Updated the application structure for improved routing, theming, and authentication.

- **Bug Fixes**
  - Addressed various internal issues related to component rendering and state management.

- **Documentation**
  - Expanded documentation to include new components and usage instructions.

### CODE REVIEW

File: src/components/MapAndTableControls/MapAndTableControls.jsx [hunk 1]
```diff
@@ -91,29 +91,6 @@ const MapAndTableControls = ({ map = undefined, view, setView }) => {
...
-      const { latitude, longitude } = foundSampleEvent
-      map.setCenter([longitude, latitude])
-      map.setZoom(18)
-    }
-  }
-
-  const hasSelectedSite = () => {
-    return queryParams.has('sample_event_id')
-  }
-
```
Comment (mmnoo) [inline src/components/MapAndTableControls/MapAndTableControls.jsx:116]: unused code. Missed linting for previous work.

**Review by alanjleonard — CHANGES_REQUESTED** [review]
Need to zoom so the cluster of points fills the map extent (excluding the panes). On the new dashboard, if you click on a cluster, it sometimes just splits into two and zooms a tiny bit. We want it to work like the old dashboard where it zooms in much more. 

In the screen cast:
- Old dashboard on the left, lots of zoom when clicking a marker (good)
- New dashboard on the right, very little zoom when clicking a marker (bad)

https://github.com/user-attachments/assets/ae2fc737-441c-4957-9118-4d401c33d9a8

**Comment by mmnoo** [comment]
@alanjleonard I made is zoom in more. Let me know if its better.

**Comment by mmnoo** [comment]
[Adjust cluster sizes and cluster onClick zoom level](https://trello.com/c/VUJUeWdz/1009-adjust-cluster-sizes-and-cluster-onclick-zoom-level)

File: src/components/MaplibreMap.jsx [hunk 2]
```diff
@@ -229,7 +229,7 @@ const MaplibreMap = ({ view, setView }) => {
 
         mapRef.current.easeTo({
           center: features[0].geometry.coordinates,
-          zoom,
+          zoom: zoom + 2,
         })
```
[the remark this answers is not in the document]
Comment (mmnoo) [reply src/components/MaplibreMap.jsx:233]: Hmm, good idea. Ill put something in the backlog for that.

**Pull request merged**